CONTRIBUTING.md

```diff
@@ -1,60 +1,85 @@
+feedback and quicker turnaround. You can also add pull requests to bug reports
+at [bugs.php.net](http://bugs.php.net/).
+
 PHP welcomes pull requests to [add tests](#writing-tests), fix bugs and to
 implement RFCs. Please be sure to include tests as appropriate!
 
 If you are fixing a bug, then please submit your PR against the lowest actively
 supported branch of PHP that the bug affects (only green branches on
 [the supported version page](http://php.net/supported-versions.php) are supported).
 For example, at the time of writing in mid-2017, the lowest supported version is
-PHP 7.0, which corresponds to the `PHP-7.0` branch in Git. Please also make sure you
-add a link to the PR in the bug on [the bug tracker](https://bugs.php.net/).
+PHP 7.0, which corresponds to the `PHP-7.0` branch in Git. Please also make sure
+you add a link to the PR in the bug on [the bug tracker](https://bugs.php.net/).
 
-Pull requests implementing RFCs should be submitted against `master`.
+Pull requests implementing RFCs should be submitted against `master` branch.
 
 Pull requests should *never* be submitted against `PHP-x.y.z` branches, as
 these are only used for release management.
 
 If your pull request exhibits conflicts with the base branch, please resolve them
 by using `git rebase` instead of `git merge`.
 
-## Filing bugs
-
-Bugs can be filed on the [PHP bug tracker](https://bugs.php.net/). If this is
-the first time you've filed a bug, we suggest reading the
-[guide to reporting a bug](https://bugs.php.net/how-to-report.php).
-
-Where possible, please include a self-contained reproduction case!
-
-## Feature requests
-
-Feature requests are generally submitted in the form of
-[Requests for Comment](https://wiki.php.net/rfc/howto), ideally accompanied by
-[pull requests](#pull-requests). You can find the extremely large list of RFCs
-that have been previously considered on the
-[PHP Wiki](https://wiki.php.net/rfc).
-
-You may want to read
-[The Mysterious PHP RFC Process](https://blogs.oracle.com/opal/entry/the_mysterious_php_rfc_process)
-for additional notes on the best way to approach submitting an RFC.
+| Branch            |                    |
+| ----------------- | ------------------ |
+| master            | The active development branch. |
+| PHP-7.2           | For releasing the PHP 7.2.x series. This is a prerelease version. |
+| PHP-7.1           | For releasing the PHP 7.1.x series. This is a current stable version and is open for bugfixes only. |
+| PHP-7.0           | For releasing the PHP 7.0.x series. This is a current stable version and is open for bugfixes only. |
+| PHP-5.6           | For releasing the PHP 5.6.x series. This is a current stable version and is open for security bugfixes only. |
+| PHP-5.5 and below | These branches are closed. |
+| PHP-X.Y.Z         | These branches are used for the release managers for tagging the releases, hence they are closed to the general public. |
```

Comment:
Not sure if this list should be here as it mandates some maintenance that is already done elsewhere. Maybe give only examples with `X.Y.Z`

Reply:
Table with branches removed since the text above is probably more descriptive, yes. And more future prone.